docker/db/init

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+# customize postgres startup here
+# 
+# Synopsis: /init secret [username]
+#  or starting current state: /init
+
+if [ "$#" -ge 1 ]; then
+  SECRET=$1
+fi
+if [ "$#" -ge 2 ]; then
+  DBUSER=$2
+elif [ "$#" -eq 1 ]; then
+  DBUSER=root
+fi
+echo 'host all all 0.0.0.0/0 md5' >> /etc/postgresql/9.1/main/pg_hba.conf
+
+# start the server (background it)
+su postgres sh -c "/usr/lib/postgresql/9.1/bin/postgres -D /var/lib/postgresql/9.1/main  -c config_file=/etc/postgresql/9.1/main/postgresql.conf -c listen_addresses=*" &
+
+sleep 10  # make sure it is all the way up
+
+# run on db create and remove
+UUE_SCRIPT='/tmp/script/template_uuen.sql'
+if [ -n "${SECRET}" -a -f ${UUE_SCRIPT} ]; then
+  # create rails user
```

Comment:
We also create a database with the same name as the user.  It seems like logins can fail unless they specify the database to connect too and this database doesn't exist.

The UUE_SCRIPT is from some Q&A site, but I can't find the link right now.